lib/tasks/tests-task.ts

```diff
@@ -0,0 +1,27 @@
+import { ITask, ICheckupResult, IProject } from '../../interfaces';
+import Task from '../task';
+import FileSearcher from '../utils/file-searcher';
+
+const SEARCH_PATTERNS = {
+  unit: ['**/tests/unit/**/*.js'],
+  acceptance: ['**/tests/acceptance/**/*.js'],
+  integration: ['**/tests/integration/**/*.js'],
+};
+
+export default class TestsTask extends Task implements ITask {
```

Comment:
I wonder if we should have a generic `FileSearchTask`? This and the type task look almost identical, apart from the search patterns they use and the property they store their result in. Thoughts?

Reply:
great point! makes sense to have a common class that does the globby searches